recipes/botan/all/conanfile.py

```diff
@@ -72,8 +72,9 @@ def source(self):
         os.rename(extracted_dir, "sources")
 
     def build(self):
-        for patch in self.conan_data["patches"][self.version]:
-            tools.patch(**patch)
+        if "patches" in self.conan_data:
```

Comment:
another good option ... 

in another PR I saw:

```yml
# config.yml

patches:
  "2.13.0":
    []
```

Reply:
Do you mean `all/conandata.yml`, or can I _also_ configure patches in `config.yml`? In any case, I considered that, but found it might be annoying to add this for any future version that does not contain a patch. The recipes for glm and sqlitecpp also to the `if "patches"` way.

Would you prefer me to change it?

Reply:
Sorry, I mean conandata.yml. No problem, keep your design, it's another good alternative as well.

Reply:
this was already in place but is unnecessary. Let's keep it as is!